articles/quickstart/backend/ruby/01-authentication.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Authentication
```

Comment:
Let's rename this to Authorization.

Please also change the filename for this md file to `01-authorization` and make sure to use this name in the `index.yml` file.